<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadasLab</class>
 <widget class="QDialog" name="CadasLab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>536</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>459</width>
    <height>536</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>459</width>
    <height>536</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CONTROLE DE PATRIMONIO</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Departamento de Mecatrônica - 2017</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CEFET-MG</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CADASTRAR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Laboratório</string>
   </property>
   <widget class="QLineEdit" name="lab_setCodigo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Código</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lab_setProfessor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="placeholderText">
     <string>Prof. Responsável</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lab_setTecnico">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Técnico do Laboratório</string>
    </property>
   </widget>
   <widget class="QPushButton" name="limpar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_confirma">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lab_setLocalizacao">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Localização</string>
    </property>
   </widget>
   <widget class="QLabel" name="fail">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(217, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="lab_setDescricao">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Descrição</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/img/img/voltar.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>limpar</sender>
   <signal>clicked()</signal>
   <receiver>lab_setTecnico</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>limpar</sender>
   <signal>clicked()</signal>
   <receiver>lab_setProfessor</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>limpar</sender>
   <signal>clicked()</signal>
   <receiver>lab_setLocalizacao</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>limpar</sender>
   <signal>clicked()</signal>
   <receiver>lab_setCodigo</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>CadasLab</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
